samples/drivers/led_lp503x/README.rst

```diff
@@ -0,0 +1,36 @@
+.. _lp503x:
+
+LP5030/6: 10 or 12 RGB channels
+###############################
+
+Overview
+********
+
+This sample controls up to 12 LEDs connected to a LP503x driver.
+
+First, for each LED information is retrieved using the led_get_info syscall
+and printed in the log messages. Next, from an infinite loop, a test pattern
+(described below) is applied to all the LEDs simultaneously (using the
+led_write_channels syscall) and then to each LED one by one (using the
+led_set_{brightness,color} syscalls).
+
+Test pattern:
+
+ For each color in red green blue white yellow purple cyan orange:
+  - set the color
+  - turn on
+  - turn off
+  - set the brightness gradually to the maximum level
+  - turn off
```

Comment:
Did you really mean to indent the "For each [...]" paragraph? That creates a [definition list](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html). The output looks like this:

![lp503x-docs](https://user-images.githubusercontent.com/62298616/89227701-faed0400-d592-11ea-9279-13e71fdfc413.png)

If that is really what you wanted, OK, but otherwise you might want to tweak this.

Reply:
Thanks for spotting this. It is fixed in the next version.